experimental/LieAlgebras/src/serialization.jl

```diff
@@ -314,13 +314,12 @@ end
 @register_serialization_type WeylGroup uses_id
 
 function save_object(s::SerializerState, W::WeylGroup)
-  save_data_dict(s) do
-    save_typed_object(s, root_system(W), :root_system)
-  end
+  RS_ref = save_as_ref(s, root_system(W))
+  save_object(s, RS_ref)
```

Comment:
Since weyl groups may potentially get more serializable properties, I would really like to keep the dict with an explicit label here

Reply:
ya sorry, this was a regression